### ISSUE TITLE
[ERROR] The example script for agent_motion_prediction possibly has issues

### ISSUE THREAD

**Issue by giaminhhoang** [lyft/l5kit#38]
Hi,

I tried to run the example scripts. While `visualise_data` worked well, the `agent_motion_prediction`  first has strange eval_dataloader as follows:
`dm = LocalDataManager(None)
# ===== INIT DATASETS
rasterizer = build_rasterizer(cfg, dm)
train_dataloader = build_dataloader(cfg, "train", dm, AgentDataset, rasterizer)
eval_dataloader = build_dataloader(cfg, "val", dm, AgentDataset, rasterizer)`

This yielded the size of `eval_dataloader` is zero which led to error at the evaluation (training was fine).

I tried change the `scene_indices` for `val_data_loader` to 1 in the file `agent_motion_config` and got non-zero `eval_dataloader` and the result below
```
+-------------+--------+
| future step |  MSE   |
+-------------+--------+
|      1      |  2.30  |
|      2      |  7.10  |
|      3      | 54.06  |
|      4      | 106.99 |
|      5      | 134.82 |
|      6      | 205.89 |
|      7      | 181.60 |
|      8      | 192.88 |
|      9      | 198.58 |
|      10     | 225.66 |
...
```
However, the `Visualise Results` doesn't work.
`---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
 in 
     28 
     29 # convert coordinates to AV point-of-view so we can draw them
---> 30 predicted_positions = transform_points(np.concatenate(predicted_positions), data_ego["world_to_image"])
     31 target_positions = transform_points(np.concatenate(target_positions), data_ego["world_to_image"])
     32 

<__array_function__ internals> in concatenate(*args, **kwargs)

ValueError: need at least one array to concatenate`

Best.

**Comment by lucabergamini** [comment]
whoops, we messed up with the config file during refactoring. It will be solved by noon :)

no eval data:
- scene indices set to -1 will solve it by picking the entire dataset (note you're training and testing on the same dataset, to try a full proper train change the `val_data_loader/datasets/key` to something else than `sample.zarr`)

error in visualisation:
- that happens when a frame doesn't have any agent in it. I'm changing the code to pick the first frame with at least one agent

**Comment by lucabergamini** [comment]
fixed in #40
